mock_db.sh

```diff
@@ -0,0 +1,26 @@
+# This script handles local database creation to make testing easier.
+# Run with `make db`
+
+postgres -V
+if [ "$?" -gt "0" ]; then
+  echo "MAKE: Postgres not installed!"
+  exit 1
+fi
+
+echo "MAKE: Killing existing postgres processes..."
+pg_ctl -D /usr/local/var/postgres stop -s -m fast
+pg_ctl -D /usr/local/var/postgres start
+while true; do
+  pg_ctl -D /usr/local/var/postgres status
+  if [ "$?" -gt "0" ]; then
+    echo "MAKE: Waiting for postgres to start..."
+    sleep 3
+  else
+    break
+  fi
+done
+
+createuser -s rocket_test
+createdb rocket_test_db
+psql -d rocket_test_db -a -f ./schema/tables.sql
+echo "MAKE: Local database ready to go!"
```

Comment:
I think we could get rid of most of this logic by just containerizing our DB for tests. Also, we don't need to wait for the DB to start with a script like this, go-pg does it for you: https://github.com/ubclaunchpad/rocket/blob/master/data/dal.go#L24.